src/containers/AccessPointDetails/components/General/index.js

```diff
@@ -0,0 +1,1060 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { Card, Form, Input, Table, Collapse, Select, Tooltip } from 'antd';
+import { InfoCircleOutlined } from '@ant-design/icons';
+import Button from 'components/Button';
+import styles from '../../index.module.scss';
+
+const { Item } = Form;
+const { Option } = Select;
+const { Panel } = Collapse;
+
+const General = ({ data }) => {
+  const [form] = Form.useForm();
+
+  const layout = {
+    labelCol: { span: 5 },
+    wrapperCol: { span: 14 },
+  };
+
+  const handleOnSave = () => {
+    form.validateFields().catch(() => {});
+  };
+
+  useEffect(() => {
+    form.setFieldsValue({
+      enableRadio2dot4: data.details.advancedRadioMap.is2dot4GHz.radioAdminState,
+      enableRadio5U: data.details.advancedRadioMap.is5GHzU.radioAdminState,
+      enableRadio5L: data.details.advancedRadioMap.is5GHzL.radioAdminState,
```

Comment:
As I said below; radioMaps are dynamic and need to be able to handle any combo of radios, some APs may only just have is2dot4GHz